pkg/docker/config/config.go

```diff
@@ -232,6 +240,9 @@ func modifyJSON(sys *types.SystemContext, editor func(auths *dockerConfigFile) (
 		}
 	}
 
+	if legacyFormat {
+		return fmt.Errorf("writes to auth.json based on legacy format docker config like %s is not supported", path)
```

Comment:
(Non-blocking:
- `writes… is not` is inconsistent
- could just say `writes to %s using legacy format` without mentioning the `auth.json` file name that is not all that likely to be used in this case.
)

Reply:
updated error message to `writes to %s using legacy format` @mtrmac

Reply:
…`are not supported`, I shouldn’t have been lazy. Sorry.

Reply:
no worries ... update pushed